Source/RunActivity/Viewer3D/RollingStock/MSTSLocomotiveViewer.cs

```diff
@@ -1560,6 +1560,7 @@ public interface ICabViewMouseControlRenderer
         bool IsMouseWithin();
         void HandleUserInput();
         string GetControlName();
+        string GetControlLabel();
```

Comment:
You should use a property with a getter in the interface instead of a getter method.

Reply:
What is the advantage? There is a GetControlName() since years above it.

Reply:
Using the getter syntax makes code simpler.
And repeating what is done in the old code doesn't make it better. It will probably be refactored later.

Reply:
Getter replaed with property now.